recipe/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
+cd build
+
+cmake -G "NMake Makefiles" ^
+      -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% ^
+      -D CMAKE_BUILD_TYPE=Release ^
+      ..
+if errorlevel 1 exit 1
+
+nmake
+if errorlevel 1 exit 1
+
```

Comment:
I know we are skipping the testing step on *NIXes (forgetting why ATM), but I imagine we will do the same on Windows as you have done here. Though it is worth noting that testing can occur with `ctest` or sometimes something like `nmake test` or `nmake check`.